docs/components/form-associated.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Form-Associated Components
+sidebar_label: Form-Associated Components
+description: Form-Associated Stencil Components
+slug: /form-associated
+---
+
+# Building Form-Associated Components in Stencil
+
+As of $VERSION Stencil has experimental support for form-associated custom
```

Comment:
I feel _pretty confident_ this'll be 4.5.0. If that's not the case, that's fine -d just adding this comment so we don't forget.
```suggestion
As of 4.5.0 Stencil has experimental support for form-associated custom
```

Reply:
I'll misspell `VERSION` on purpose so we can't merge till we update